autode/transition_states/base.py

```diff
@@ -15,11 +15,22 @@ class TSbase(Species):
 
     def _init_graph(self):
         """Set the molecular graph for this TS object from the reactant"""
-        logger.warning(f'Setting the graph of {self.name} from reactants')
+        if self.reactant is not None:
+            logger.warning(f'Setting the graph of {self.name} from reactants')
+            self.graph = self.reactant.graph.copy()
+
```

Comment:
I'm not entirely sure what this change is for yet, but is it a problem that it could have reactant and atoms, so reset the graph twice?

Reply:
Good point – should be if reactant is None and self.atoms is None